### ISSUE TITLE
What is the best way to convert numeric excel sheet into a matrix ?

### ISSUE THREAD

**Issue by asavasci** [felipenoris/XLSX.jl#124]
I am curious if there is a direct way to convert a numeric data from excel sheet into a matrix/vector because Worksheet data type is not allowed to direct mathematical operations. I am facing the the error messages when I want to convert Worksheet into matrix below 

```
xf = XLSX.readxlsx(file)
G = convert(Matrix, xf["sheet"])
```
MethodError: Cannot `convert` an object of type XLSX.Worksheet to an object of type Array{T,2} where T

Another way I tried also gave error below

`G = convert(Matrix, DataFrame(xf["sheet"]))`
ArgumentError: no default `Tables.columns` implementation for type: XLSX.Worksheet
columns at fallbacks.jl:180 [inlined]

`G = convert(matrix, XLSX.readtable(file, "sheet"))`
ERROR: AssertionError: Column labels must be unique.

**Comment by felipenoris** [comment]
Please refer to [XLSX.readdata](https://felipenoris.github.io/XLSX.jl/stable/api/#XLSX.readdata) and [XLSX.getdata](https://felipenoris.github.io/XLSX.jl/stable/api/#XLSX.getdata).

Currently, these methods return a Matrix of `Any` type, since a worksheet cell can have any type and these methods do not apply type inference, so you may have to further convert the result to the desired data type, as in the following example.

```julia
julia> m = XLSX.readdata("data/numbers.xlsx", "float!A1:B4")
4×2 Array{Any,2}:
  1.2  1000.0
 10.0  1000.0
 10.0     1.0
 10.0     1.0

julia> m2 = convert(Matrix{Float64}, m)
4×2 Array{Float64,2}:
  1.2  1000.0
 10.0  1000.0
 10.0     1.0
 10.0     1.0
```

**Comment by asavasci** [comment]
Thanks, that's working, however, for large-sized matrices tracking the sheet range may not be efficient. What would you recommend for this?

**Comment by felipenoris** [comment]
If you know the sheet range, this method is the most efficient.

**Comment by felipenoris** [comment]
Closed by 84fe5573073c32738ed6fb712738d830dbfb5b7c.

**Comment by felipenoris** [comment]
Check out this example.

```julia
julia> columns, labels = XLSX.readtable("myfile.xlsx", "mysheet")
(Any[Any[1, 2, 3], Any["first", "second", "third"]], Symbol[:HeaderA, :HeaderB])
julia> m = hcat(columns...)
3×2 Array{Any,2}:
 1  "first"
 2  "second"
 3  "third"
```
